strawberry/cli/commands/import_schema.py

```diff
@@ -0,0 +1,64 @@
+import re
+from typing import List
+
+import click
+
+from graphql.error import GraphQLError
+from graphql.language import parse
+
+from ...utils.sdl_transpiler import render_template
+
+
+# Regex pattern to extract import statement
+GQL_IMPORT_REGEX = r"#import \"(.*\.gql|.*\.graphql)\""
+GQL_SCHEMA_TYPE = r"(?si)^Schema.*?}"
+
+
+@click.command("import-schema", short_help="Parses SDL file to python dataclasses")
```

Comment:
shouldn't `"Parses SDL file to python dataclasses"` be something more like: `"Parses SDL file to strawberry types"`?

Reply:
This one was easy to fix :D